.github/workflows/main.yaml

```diff
@@ -4,11 +4,31 @@ on:
   push:
     branches:
       - maintenance-3.0.x
-  pull_request:
 
 jobs:
+  lint:
+    uses: nuxeo/nuxeo-elements/.github/workflows/lint.yaml@ELEMENTS-1459-pr-check-split
+    secrets:
+      NPM_PACKAGES_TOKEN: ${{ secrets.NPM_PACKAGES_TOKEN }}
+
+  test:
+    uses: nuxeo/nuxeo-elements/.github/workflows/test.yaml@ELEMENTS-1459-pr-check-split
+    secrets:
+      NPM_PACKAGES_TOKEN: ${{ secrets.NPM_PACKAGES_TOKEN }}
+      SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
+    with:
+      branch: maintenance-3.0.x
+
+  storybook:
+    uses: nuxeo/nuxeo-elements/.github/workflows/storybook.yaml@ELEMENTS-1459-pr-check-split
```

Comment:
this needs to be changed before merging the code, and should reference `lint.yaml@maintenance-3.0.x`